documentation/specs/Banner.md

```diff
@@ -0,0 +1,46 @@
+# Banner Component Specification
+
+## Overview
+
+The `Banner` component displays a prominent message above the navigation bar at the top of the page that cannot be dismissed.
+
+### Use Cases
+
+The primary use case of the `Banner` component is to inform the user of promotions and other offerings on the marketing (logged out) site. **Not** to be used to call attention to the user as a result of interacting with the UI.
+
+---
```

Comment:
didnt include prior art from polaris as i think our use case does differ a bit, but open to including it and noting the differences